packages/react/package.json

```diff
@@ -67,7 +67,7 @@
     "lodash": "4.17.21",
     "mapbox-gl": "1.13.1",
     "maplibre-gl": "2.1.9",
-    "maplibre-gl-js-amplify": "2.0.4",
+    "maplibre-gl-js-amplify": "2.1.0",
```

Comment:
nit: we should still make a changeset because this changes customer surface area

Reply:
Done. Added changeset.